packages/libs/components/src/components/Spinner.tsx

```diff
@@ -1,6 +1,10 @@
 import CircularProgress from '@material-ui/core/CircularProgress';
 
-export default function Spinner() {
+interface Props {
+  size?: string | number;
+}
+
+export default function Spinner({ size }: Props) {
```

Comment:
Shouldn't `size` default to 50 for backwards compatibility?

Reply:
Whoops, it totally did until I accidentally deleted it! Thanks for catching that! Restored in: https://github.com/VEuPathDB/web-monorepo/pull/232/commits/d0a319a27ddd17dfc0aafc2eb2aea71b5d20d63f